Specs/Scene/ModelExperimental/InstancingPipelineStageSpec.js

```diff
@@ -407,6 +414,7 @@ describe("Scene/ModelExperimental/InstancingPipelineStage", function () {
       const shaderBuilder = renderResources.shaderBuilder;
       const uniformMap = renderResources.uniformMap;
       const runtimeNode = renderResources.runtimeNode;
+      renderResources.model.sceneGraph.components = components;
```

Comment:
I'm not familiar with the i3dm loader so I'm just curious, why is this line necessary?

Reply:
oh it's that the render resources are being mocked here, so we need to insert the loaded components. I'll leave a comment explaining this.